tests/suite/mod.rs

```diff
@@ -4,10 +4,10 @@
 pub use super::test_markdown_html;
 
 mod footnotes;
```

Comment:
Not sure why rustfmt wanted to do this, did it start reordering imports recently?

Should I revert this or just leave it?

Reply:
It's fine :-)